internal/zinc-persist/src/main/scala/sbt/internal/inc/binary/converters/ProtobufReaders.scala

```diff
@@ -26,11 +27,17 @@ final class ProtobufReaders(mapper: ReadMapper) {
     java.nio.file.Paths.get(path).toFile
   }
 
+  private def readLong(bs: ByteString) = {
```

Comment:
It's not possible to use `ByteBuffer` because `bytes` in protobuf relies on `ByteString`. We cannot change this data type before.